packages/web/src/pages/saved-page/components/desktop/LibraryCategorySelectionMenu.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  savedPageActions,
+  savedPageSelectors,
+  LibraryCategory,
+  LibraryCategoryType
+} from '@audius/common'
+import { HarmonySelectablePill } from '@audius/stems'
+import { useDispatch, useSelector } from 'react-redux'
+
+import styles from './LibraryCategorySelectionMenu.module.css'
+
+const { getSelectedCategory } = savedPageSelectors
+const { setSelectedCategory } = savedPageActions
+
+const CATEGORIES = [
+  {
+    label: 'All',
+    value: LibraryCategory.All
+  },
+  {
+    label: 'Favorites',
+    value: LibraryCategory.Favorite
+  },
+  {
+    label: 'Reposts',
+    value: LibraryCategory.Repost
+  },
+  {
+    label: 'Purchased',
+    value: LibraryCategory.Purchase
+  }
+]
+
+export const LibraryCategorySelectionMenu = () => {
+  const dispatch = useDispatch()
+  const selectedCategory = useSelector(getSelectedCategory)
+  const handleClick = (value: LibraryCategoryType) => {
+    dispatch(setSelectedCategory(value))
+  }
+
+  return (
+    <div role='radiogroup' className={styles.container}>
+      {CATEGORIES.map((c) => (
+        <HarmonySelectablePill
```

Comment:
Planning to add a SelectablePillRadioGroup to Harmony + allow specifying HarmonySelectablePill element type as radio input. It's not the most straightforward so gonna do it as a project follow-on most likely.